install.sh

```
@@ -0,0 +1,119 @@
+}
+
+download_and_verify() {
+  target="$1"
+  CREATED_TMPDIR=0
+  if [ "${AXON_INSTALL_TMPDIR:-}" ]; then
+    tmpdir="$AXON_INSTALL_TMPDIR"
+    case "$tmpdir" in
+      ""|"/") fail "unsafe AXON_INSTALL_TMPDIR: $tmpdir" ;;
+    esac
+    [ -d "$tmpdir" ] || fail "AXON_INSTALL_TMPDIR must be an existing directory"
+    [ -O "$tmpdir" ] || fail "AXON_INSTALL_TMPDIR must be owned by the current user"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, test -O is undefined.

(SC3067)